samples/bot-batch-operations/nodejs/assets/sample.json

```diff
@@ -35,8 +35,8 @@
       {
         "type": "Image",
         "order": 100,
-        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/bot-conversation/nodejs/Images/ConversationBot.gif",
-        "alt": "Solution UX showing conversation events for teams bot"
+        "url": "https://raw.githubusercontent.com/southworks/Microsoft-Teams-Samples/bf92bc9c53ed366037c70c51af8432782575cecf/samples/bot-batch-operations/nodejs/Images/GetFailedEntries.png",
```

Comment:
Don't use the SW fork. Leave the main branch URL even when the sample does not exist yet.